packages/@okta/vuepress-site/docs/release-notes/2020-07-2/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Okta API Products Release Notes
+---
+
+## 2020.07.2
+
+| Change                                                      | Expected in Preview Orgs |
+| ----------------------------------------------------------- | ------------------------ |
+| [Bugs fixed in 2020.07.2](#bugs-fixed-in-2020-07-2)         | July 29, 2020            |
```

Comment:
nit: bug is singular this week (annoying, I know).

Reply:
keep in mind the anchor link will change too

Reply:
Fixed - thanks!